tests/queries/0_stateless/03131_hilbert_coding.sql

```diff
@@ -0,0 +1,48 @@
+SELECT '----- START -----';
```

Comment:
Would be good to add some test cases regarding handling exceptions. You can do it in this file by writing


```
query_that_raises_exception; -- { serverError your_error_code }
```